pipeline/outputs/postgresql.md

```diff
@@ -56,6 +56,7 @@ Make sure that the `fluentbit` user can connect to the `fluentbit` database on t
 | `Password` | Password of PostgreSQL username | - |
 | `Database` | Database name to connect to | - \(current user\) |
 | `Table` | Table name where to store data | - |
+| `Connection_Options` | PostgreSQL connection options | - |
```

Comment:
Any restrictions on content? Should we quote anything with spaces in, etc?

Reply:
no, it allows string with spaces.

ex: passing multiple options:

```
Connection_Options -c geqo=off -c statement_timeout=0
```

[pg connection option ref](https://www.postgresql.org/docs/devel/libpq-connect.html#LIBPQ-CONNECT-OPTIONS)

Reply:
I was updated. Could you check again please?